sql/analyzer/parallelize.go

```diff
@@ -72,8 +72,11 @@ func parallelize(ctx *sql.Context, a *Analyzer, node sql.Node, scope *plan.Scope
 		return node, transform.SameTree, nil
 	}
 
-	node, same, err := transform.NodeWithCtx(node, nil, func(c transform.Context) (sql.Node, transform.TreeIdentity, error) {
-		if !isParallelizable(c.Node) {
+	foundOrderedDistinct := false
+	newNode, same, err := transform.NodeWithCtx(node, nil, func(c transform.Context) (sql.Node, transform.TreeIdentity, error) {
+		if _, ok := c.Node.(*plan.OrderedDistinct); ok {
```

Comment:
Can't this go in isParallelizable?

Reply:
That's what i tried first, but we check bottom-up. Nested children are still parallelized there.